app/components/layout/sidebar_component.html.erb

```diff
@@ -120,7 +117,7 @@
           <h3 class="py-2 pl-4 font-semibold text-slate-900 dark:text-white">
             <%= t(:"general.default_sidebar.title") %>
           </h3>
-          <ul class="py-2 w-full text-sm text-slate-700 dark:text-slate-200">
+          <ul class="py-2 w-full text-sm text-slate-700 dark:text-slate-200" role="menu">
```

Comment:
I think you need to have an aria label here, maybe `aria-label="<%= t(:"general.default_sidebar.title")`, or I guess you could add an id the the h3 above and do aria-labelledby.

Reply:
Refactored in a5ec8c7